.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
+_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_
```

Comment:
Just leaving a comment here but I think I'd rather put a `x` inside the box or remove the line when not applicable, but I might be the black sheep here 😄

Reply:
Just explaining my chain of thoughts. But doesn't mean it's the best choice:
- The issue with removing the line is that contributors can remove the line when it should stay.
- Put an "x" inside the box isn't really true.

Best option IMO would be to surround with `~` but nobody will understand how it works.

Reply:
NA is fine, since we so know that the developer read the info...

Reply:
So let's go this way, I'll try to be a kind sheep then 😅